.circleci/config.yml

```diff
@@ -90,6 +90,17 @@ jobs:
             - v1-gopkg-cache-{{ arch }}-{{ checksum "Gopkg.lock" }}
             - v1-gopkg-cache-{{ arch }}-{{ .Branch }}
 
+      - run:
+          name: Install operator-sdk
+          command: |
+              mkdir -p ${GOPATH}/src/github.com/operator-framework
+              cd ${GOPATH}/src/github.com/operator-framework
+              git clone https://github.com/operator-framework/operator-sdk
```

Comment:
if you want to save `git checkout`, `git clone` has --branch flag where tags are also taken. Up to you, it is ok also to do clone and checkout.

Reply:
Changed. Using --branch leaves the status in HEAD dettached state, but it's not a problem if using it only on the CI because it is not going to be modified.